CP_Source/Apps/Frame/README.md

```diff
@@ -109,28 +231,34 @@ General
+
+
 ### Basic Frame App Profile
+**IGEL UMS-11 command:**
 ```js
-import 'igel/frame-app-basic-profile.xml'
+import 'igel/frame-app-basic-profile-ums11.xml'
 ```
 
 This Custom Profiles simply enables a Frame App icon on the IGEL Desktop. To launch Frame App with certain command line arguments, edit IGEL profile and append your command line argument(s) to the end of the command line.
 
-Below is a table of [Linux command-line arguments for Frame App](https://docs.fra.me/platform/session/frame-app/config/?operating-systems=linux#frapp-args):
-
-| Command Line Argument   | Description                                                                                                                                                                                                                                                    | Syntax                              |
-| ----------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------- |
-| `disable-mouse-lock`    | By default, Frame App will ignore mouse forward/back button events. If this argument is used, Frame App will respond to mouse forward/back button events. When a user is in session, these mouse events can cause the user to exit their session accidentally. | `./Frame --disable-mouse-lock`      |
-| `displays-auto-arrange` | Frame App will launch with virtual displays configured to match your local environment.                                                                                                                                                                        | `./Frame --displays-auto-arrange`   |
-| `kiosk`                 | Instructs Frame App to launch in full screen, a.k.a. “Kiosk mode.”                                                                                                                                                                                             | `./Frame --kiosk`                   |
-| `url`                   | Designates the startup URL.                                                                                                                                                                                                                                    | `./Frame --url=console.nutanix.com` |
-| `x11-window`            | Switch from GTK (default) to X11 Windows. This argument should be used with HP ThinPro OS clients.                                                                                                                                                             | `./Frame --x11-window`              |
+Command line arguments for Frame app are defined here: [Linux command-line arguments](https://docs.fra.me/platform/session/frame-app/config/?operating-systems=linux#frapp-args).
```

Comment:
Should be:
https://docs.dizzion.com/platform/session/frame-app/config#command-line-arguments

Reply:
Linux is under its own tab